.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+
+sudo: false
+
+cache:
+    yarn: true
+    directories:
+        - node_modules
+
+env:
+    global:
+        - NODE_ENV=test
+        - TZ=Europe/Paris
+
+node_js:
+    - "8.10"
```

Comment:
v10 is LTS now

Reply:
Yep but AWS Lambda only run 8.10 as max, not 10 yet.